src/resources/resources.json

```diff
@@ -1168,7 +1171,10 @@
       "internalCdn": "Internes CDN",
       "externalUrl": "Externe URL",
       "internalUrl": "Interne URL",
-      "copyrightInfos": "Copyright Infos"
+      "copyrightInfos": "Copyright Infos",
+      "testsOrder": "Anzeige Reihnfolge",
+      "testsOrderGiven": "als gegeben",
```

Comment:
You didn't run gulp after changing it in YAML to "wie angegeben"

Reply:
Stimmt! I have now, thanks